drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -265,6 +265,9 @@ static uint32_t attach_pin_to_isr(uint32_t port, uint32_t pin, uint32_t isr_no)
 #if (FSL_FEATURE_PINT_NUMBER_OF_CONNECTED_OUTPUTS > 8)
 	#error having more than 8 PINT IRQs not supported in driver
 #elif (FSL_FEATURE_PINT_NUMBER_OF_CONNECTED_OUTPUTS > 4)
+#ifdef CONFIG_SOC_SERIES_LPC51U68
+	pint_idx = isr_no - PIN_INT0_IRQn;
+#endif
```

Comment:
Please delete the above 3 lines as LPC51U68 has less than 4 outputs and should never go into this block of code.